src/GameObject.cpp

```diff
@@ -79,29 +79,11 @@ void GameObject::setYAxisRotation(float angle) {
 void GameObject::calculateAndSetInitialRotation() {

```

Comment:
I'm thinking we should probably put this orientation code in each objects render component. Since the bunnies will likely have a different initial orientation than other things.